opencost/src/settings.tsx

```diff
@@ -0,0 +1,85 @@
+import { NameValueTable } from '@kinvolk/headlamp-plugin/lib/components/common';
+import Box from '@mui/material/Box';
+import Checkbox from '@mui/material/Checkbox';
+import MenuItem from '@mui/material/MenuItem';
+import Select from '@mui/material/Select';
+import TextField from '@mui/material/TextField';
+
+export function Settings(props) {
+  const { data, onDataChange } = props;
+
+  const settingsRows = [
+    {
+      name: 'Enable in list view',
+      value: (
+        <Checkbox
+          checked={data?.isEnabledInListView}
+          onChange={(event: React.ChangeEvent<HTMLInputElement>) =>
+            onDataChange({ ...data, isEnabledInListView: event.target.checked })
+          }
+        />
+      ),
+    },
+    {
+      name: 'Namespace',
+      value: (
+        <TextField
+          fullWidth
+          helperText="Enter the namespace of Opencost service"
+          defaultValue={data?.namespace ? data.namespace : ''}
+          onChange={e => onDataChange({ ...data, namespace: e.target.value })}
+          variant="standard"
+        />
+      ),
+    },
+    {
+      name: 'Service URL',
```

Comment:
Is this because the plugin places a link to that URL? If so, we need a better name and description.

Reply:
Nope this is the service URL from which Opencost API can be accessed.